src/devices/Hmc5883l/OutputRate.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Hmc5883l
+{
+    /// <summary>
+    /// HMC5883L Typical Data Output Rate (Hz)
+    /// </summary>
+    public enum OutputRate
```

Comment:
does this refer to internal reading rate? (how often it updates the internal register) If so then perhaps this should be Rate_0_75Hz
Rate_1_5Hz
Rate_75_0Hz
or something along this lines.

Also I'd remove the Unused value

@joperezr any objection to such naming convention?

Other alternatives I'm thinking of:
- Rate0Point75
- make this a property which allows setting a double value from 0 to max and round up to the closest one

Reply:
The previous code also needs to be changed, waiting for your discussion.😁

Reply:
I'm voting for double property which auto picks rounded up value

Reply:
I agree with removing the Unused value. Also, to me the best approach would be Rate_0_75Hz as that one is super clear onto what is it that we are doing and the possible values.